wordpress-mobile/WordPress-FluxC-Android
# Woo/update grouped product ids

Addresses part of woocommerce/woocommerce-android#2630 by adding the logic to update products for a grouped product.

#### Screenshots
<img src="https://user-images.githubusercontent.com/22608780/89004690-62e7e400-d320-11ea-9e9c-eee8f1eac503.png" width="300" />

#### Testing
- Run `MockedStack_WCProductsTest` and `ReleaseStack_WCProductTest`.
- Since this PR adds a migration script to add a new column to to the `WCProductModel`, it would be great to test updating the app as well.
- In the example app:
  - Click on `Woo` -> `Products` -> `Select Site`.
  - Click on `Update Product` -> `Enter product Id`. (Enter a grouped product Id).
  - Notice the grouped product Ids section at the bottom of the screen.
  - Click on `Add Product Ids` and enter a valid product Id.
  - Click on the `Tick` icon at the top of the screen.
  - Notice that the Grouped product is updated successfully.
